lib/membrane_h264_plugin/parser/nalu_parser/scheme.ex

```diff
@@ -23,7 +23,9 @@ defmodule Membrane.H264.Parser.NALuParser.Scheme do
   # * save_as_global_state_t: saves the current parser state, concerning the NALu which is
   #   currently processed, in the map under the `:__global__` key in the state. Information
   #   from the saved state can be used while parsing the following NALus.
-
+  #
+  # Furthermore, `Scheme` behavior defines `defaults/0` callback,
+  # which is used to provide the default values of the fields.
```

Comment:
Maybe just put it as a doc for `defaults` callback?

Reply:
Well, I believe it is a part of a description of how the Scheme description works - it's our private API and the other callback (`scheme/0` ) is also described in that comment